railties/test/application/per_request_digest_cache_test.rb

```diff
@@ -53,12 +54,21 @@ def index
   teardown :teardown_app
 
   test "digests are reused when rendering the same template twice" do
-    get "/customers"
-    assert_equal 200, last_response.status
+    SecureRandom.stub(:uuid, "xyz-123") do
```

Comment:
We should use a new test to test this behavior instead of changing an existing one. But better yet, why is this being tested in this file?

Reply:
Good point.  This wasn't the right spot to test.

Moved and added additional tests in `actionview/test/template/render_test.rb`